internal/api/handlers/transactions/update.go

```diff
@@ -0,0 +1,24 @@
+package transactions
+
+import (
+	"context"
+
+	"github.com/ndovnar/family-budget-api/internal/model"
+)
+
+func (t *Transactions) broadcastUpdate(ctx context.Context, updateAction model.UpdateActionType, transaction *model.Transaction) {
+	userIDs := []string{}
+
+	if transaction.FromAccountID != "" {
+		fromAccountUserIDs := t.authz.GetUserIDsHaveAccessToAccount(ctx, transaction.FromAccountID)
+		userIDs = append(userIDs, fromAccountUserIDs...)
+	}
+
+	if transaction.ToAccountID != "" {
```

Comment:
if you want to have optional parameters it's better to use pointers . For exameple: you have optional field fromAccountID, and to describe it you can use: `fromAccountID *string`. In this case you will show other developers that this is optional paramer. Check for empty string is a little confused